concourse/scripts/environment.sh

```diff
@@ -9,38 +9,22 @@ hashed_password() {
   echo "$1" | shasum -a 256 | base64 | head -c 32
 }
 
-DEPLOY_ENV=${DEPLOY_ENV:-}
+DEPLOY_ENV=${1:-${DEPLOY_ENV:-}}
```

Comment:
@alext good catch
Here you go: https://github.com/alphagov/paas-bootstrap/pull/35